endpoints/info/bidders_test.go

```diff
@@ -155,6 +127,11 @@ func TestPrepareBiddersResponseAllBaseOnly(t *testing.T) {
 			givenBidders: config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
 			expected:     `["a","c"]`,
 		},
+		{
+			name:         "mixed",
+			givenBidders: config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
+			expected:     `["a","c"]`,
+		},
```

Comment:
I think you can delete this test case. It is the same as the `alias-mixed` case on lines 126-128.

Reply:
Differentiated.